src/routes/release.js

```diff
@@ -197,15 +197,14 @@ router.get(
       return res.redirect('/');
     }
 
-    let releaseNotes = release.body;
-    const parsed = semver.parse(version);
-    if (parsed.prerelease.length) {
-      releaseNotes = releaseNotes.split(new RegExp(`@${escapeRegExp(version.substr(1))}\`?.`))[1];
-    }
-    releaseNotes =
-      '# Release Notes\n' +
-      releaseNotes.replace(/# Release Notes for [^\r\n]+(?:(?:\n)|(?:\r\n))/i, '');
+ let releaseNotes = release?.body ?? 'No release notes available.';
```

Comment:
Did you run the `prettier:write` command? This uneven whitespace seems like it would fail.